### PR TITLE
fix(scripts): release script with ssh

### PR DESCRIPTION
**Pull request by shortcuts**
## 🧭 What and Why

🎟 JIRA Ticket: -

### Changes included:

ssh via docker was broken because:
- ssh wasn't available anymore in the image
- docker keeps quotes when using the env var, so wrong credentials error 🤷🏻

### CODE REVIEW

File: scripts/common.ts [hunk 1]
```diff
@@ -246,7 +246,7 @@ export function ensureGitHubToken(): string {
   if (!process.env.GITHUB_TOKEN) {
     throw new Error('Environment variable `GITHUB_TOKEN` does not exist.');
   }
-  return process.env.GITHUB_TOKEN;
+  return process.env.GITHUB_TOKEN.replaceAll('"', '');
```
Comment (shortcuts) [inline scripts/common.ts:249]: I fixed my env var of course but lets keep this in case someone also puts them for strings

**Review by millotp — APPROVED** [review]
nice

File: scripts/package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "cleanGeneratedBranch": "ts-node ci/codegen/cleanGeneratedBranch.ts",
     "createMatrix": "ts-node ci/githubActions/createMatrix.ts",
-    "createReleasePR": "ts-node release/createReleasePR.ts",
+    "createReleasePR": "NODE_NO_WARNINGS=1 ts-node release/createReleasePR.ts",
```
[the remark this answers is not in the document]
Comment (shortcuts) [reply scripts/package.json:8]: :( idk

**Comment by shortcuts** [comment]
my rebase was broken, better now, sorry

**Pull request merged**